docs/components/cloud-console/troubleshooting/feedback-and-support.md

```diff
@@ -7,4 +7,6 @@ If you have any problems, questions, or suggestions, contact us via the [Camunda
 
 **Feedback and support** can be submitted or requested via the corresponding entry in the navigation menu. If you have a **license agreement** with us, you will be redirected to the [support queue](https://jira.camunda.com/projects/SUPPORT/) at Camunda. In the Free Plan, contact can be made using the internal form:
 
+[//]:# (This goes along with my prior note. Is this a "Trial Plan," "Free Plan," or simply (free?) trial phase? )
```

Comment:
We have two options for [plans](https://camunda.com/products/cloud#compare), but no free option at this point. We refer to this as a "Trial". For reference, I try to make sure I align with the language here - https://camunda.com/get-started

But we have to keep an eye on this as it could change. Product marketing is a good connection if you ever have questions about this and I'm not around :)